mlir/test/Dialect/Tosa/ops.mlir

```diff
@@ -580,14 +580,6 @@ func.func @test_concat(%arg0: tensor<13x21x3xf32>, %arg1: tensor<13x21x3xf32>) -
   return %0 : tensor<26x21x3xf32>
 }
 
-// -----
-// CHECK-LABEL: pad
-func.func @test_pad(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
-  %padding = tosa.const_shape {value = dense<0> : tensor<6xindex>} : () -> !tosa.shape<6>
-  %0 = tosa.pad %arg0, %padding : (tensor<13x21x3xf32>, !tosa.shape<6>) -> tensor<13x21x3xf32>
-  return %0 : tensor<13x21x3xf32>
-}
```

Comment:
nit: any reason to remove this test-case rather than add `pad_const`?

Reply:
added it back with `pad_const`